Examples/VisionExamples/Examples/ARCustomizationViewController.swift

```diff
@@ -0,0 +1,146 @@
+import MapboxVisionAR
+import UIKit
+
+/**
+ * "AR Customization" tutorial demonstrates how to customize vision AR visuals which are AR lane and AR fence.
+ * The tutorial is based on "ARNavigation" example which shows how to get the route, convert it to vision format,
+ * create vision manager anв set up the vision AR session with the route.
+ */
+class ARCustomizationViewController: ARNavigationViewController {
```

Comment:
nit:
```suggestion
class ARCustomizationExampleViewController: ARNavigationViewController {
```

Should we add "Example" suffix for all such "example"-kind VCs to track them in the future and to help to avoid a mistake when you create a dependency on them?

Reply:
such view controllers are placed in `Examples` folder which is good enough I think